### DISCUSSION TITLE
Some way to disable logging certain commands

### DISCUSSION THREAD

**Discussion by cootshk** [PaperMC/Paper#10816] [Ideas]
### Is your feature request related to a problem?

commands like `/w`, `/msg`, and `/tell` are logged in console - leaking coords to the server owner in PVP servers

### Describe the solution you'd like.

either a config option to disable logging commands, or a config option to disable logging certain commands

### Describe alternatives you've considered.

Making a plugin to override is loggable (https://bukkit.org/threads/how-can-i-prevent-the-console-from-logging-certain-commands.158152/) - this should be a config option

**Comment by cootshk** [comment]
```
[23:26:23 INFO]: Cootshk issued server command: /gamemode survival
[23:26:23 INFO]: [Cootshk: Set own game mode to Survival Mode]
```
Remove the first line, as the second line can be removed by running `/gamerule logAdminCommands false`

**Comment by electronicboy** [comment]
there is already a config option to disable general command logging inside of spigot.yml
